mobile-ui/package.json

```diff
@@ -59,13 +59,16 @@
     "@babel/core": "^7.20.0",
     "babel-plugin-transform-inline-environment-variables": "^0.4.4",
     "eslint": "^8.37.0",
+    "eslint-config-prettier": "^8.8.0",
```

Comment:
Please remove prettier out of our dependencies, we dont you it

Reply:
It may come from previous updating. Before pushing, I hope you sync code from base repository